### PR TITLE
Info: Fix /show and /requestshow interaction error

### PR DESCRIPTION
**Pull request by mia-pi-git**
https://www.smogon.com/forums/threads/bug-reports-v4-read-original-post-before-posting.3663703/page-44#post-8709275

### CODE REVIEW

File: server/chat-commands/info.ts [hunk 1]
```diff
@@ -2515,7 +2515,10 @@ export const commands: ChatCommands = {
 		if (!room.settings.requestShowEnabled) {
 			return this.errorReply(`Media approvals are disabled in this room.`);
 		}
-		if (user.can('showmedia', null, room)) return this.errorReply(`Use !show instead.`);
+		const showPerm = room.settings.permissions?.['/show'];
+		if (user.can('showmedia', null, room) && showPerm && room.auth.atLeast(user, showPerm)) {
+			return this.errorReply(`Use !show instead.`);
+		}
```
Comment (Zarel) [inline server/chat-commands/info.ts:2521]:
```suggestion
		if (user.can('showmedia', null, room, '/show')) return this.errorReply(`Use !show instead.`);
```

This is easier, right?

**Comment by Zarel** [comment]
I asked you to test the simplified version and you said it wasn't working correctly? Can you clarify?

**Comment by mia-pi-git** [comment]
> I asked you to test the simplified version and you said it wasn't working correctly? Can you clarify?

As in, `user.can('showmedia', null, room, '/show')` with the correct settings to replicate the rest of this bug, was returning false. I'll look more into it later today.

**Comment by AnnikaCodes** [comment]
What's the status of this PR?

**Comment by mia-pi-git** [comment]
> What's the status of this PR? The bug report link is no longer working, so I can't even see what the undesirable interaction was :/.

Can't repro the issue anymore. Probably fine to merge?

**Comment by AnnikaCodes** [comment]
If you can't reproduce the issue, why is the PR fixing the bug necessary?

**Comment by mia-pi-git** [comment]
> If you can't reproduce the issue, why is the PR fixing the bug necessary?

I meant that I cannot reproduce the issue that stalled the PR.

**Comment by AnnikaCodes** [comment]
Oh, okay. This is fine then, thanks!

**Pull request merged**